README.md

```diff
@@ -556,7 +556,8 @@ Note: AUR package is maintained by community, not maintainer of cargo-llvm-cov.
 
 - Branch coverage is not supported yet. See [#8] and [rust-lang/rust#79649] for more.
 - Support for doc tests is unstable and has known issues. See [#2] and [rust-lang/rust#79417] for more.
-
+- All the tests are run in a single thread (can be reproduced by running tests with `cargo test -- --test-threads=1`). This can make the test result be different from `cargo test` run if thread-dependent functions are used (like `thread_local!`). See [#208] and [rust-lang/rust#91092] for more.
```

Comment:
I think "run in a single thread" is misleading, When the user writes a multi-threaded test it will run in a multi-threaded. (`--test-threads=1`/`RUST_TEST_THREADS=1` applies only to test runners.)

I would suggest something like:

```text
- All the tests are run with `RUST_TEST_THREADS=1` to work around [rust-lang/rust#91092].
  You can pass `--test-threads` (e.g., `--test-threads=$(nproc)`) to override this behavior.
```

(The test that breaks when `RUST_TEST_THREADS=1` is used is considered to be a fundamentally fragile test, since it breaks even when the host environment is single-core, so I don't think it needs special mention here.)